app/client/lib/markdown.ts

```diff
@@ -2,6 +2,24 @@ import { sanitizeHTML } from 'app/client/ui/sanitizeHTML';
 import { BindableValue, DomElementMethod, subscribeElem } from 'grainjs';
 import { marked } from 'marked';
 
+/**
+ * Helper function for using Markdown in grainjs elements. It accepts
+ * both plain Markdown strings, as well as methods that use an observable.
+ * Example usage:
+ *
+ *    cssSection(markdown(t(`# New Markdown Function
+ *
```

Comment:
Empty line

Reply:
That's on purpose, to demonstrate new paragraphs in markdown.